Editor/Manifest/Manifest.cs

```diff
@@ -246,6 +246,18 @@ public void SetDependency(string name, string version)
             var dependency = new Dependency(name, version);
             m_Dependencies[dependency.Name] = dependency;
         }
+        
+        /// <summary>
+        /// Remove <see cref="Dependency"/> by given name (if manifest contains it)
+        /// </summary>
+        /// <param name="name">Dependency name.</param>
+        public void RemoveDependency(string name)
+        {
+            if(m_Dependencies.ContainsKey(name))
```

Comment:
Please, consider using `IsDependencyExists` method here instead of manipulating with `m_Dependencies` dictionary directly.

Reply:
Fixed.